src/store/plugins/geolocation-management.plugin.js

```diff
@@ -51,6 +52,12 @@ const handlePositionAndDispatchToStore = (position, store) => {
         position: positionProjected,
         ...dispatcher,
     })
+    if (position.coords.heading != null && position.coords.speed >= 0.5) {
+        store.dispatch('setGeolocationHeading', {
+            heading: toRadians(position.coords.heading),
+            ...dispatcher,
+        })
+    }
```

Comment:
@LukasJoss @pakb why using 0.5 here ? I understand that if the speed is 0 the heading will NaN as it cannot be computed, but why taking a 0.5 speed and not simply checking that heading is not NaN ? Why 0.5 and not any other number ? from where did this number come from ?